fe1-web/parts/lao/organizer/RollCallOpened.tsx

```diff
@@ -57,14 +66,18 @@ const RollCallOpened = () => {
       EventTags.ROLL_CALL, OpenedLaoStore.get().id.toString(),
       rollCallID, time,
     );
-    requestCloseRollCall(updateId, Array.from(attendees).map((key: string) => new PublicKey(key)))
-      .then(() => {
+
+    // Add the token of the organizer before closing the roll call
+    Wallet.generateToken(lao.id, new Hash(rollCallID)).then((token) => {
+      const attendeesList = Array.from(attendees).map((key: string) => new PublicKey(key));
+      attendeesList.push(token.publicKey);
```

Comment:
... which means you wouldn't need to convert them here.
You'd also be able to add the token from the very creation of the `attendees` state (possibly in a `useEffect` hook)

This way: 
- the default view would show that you have `1` attendee
- the number at the bottom of this screen would match the number of public keys listed once the roll call is closed
- you're not going to have `string` items representing something else :) 

Is it feasible or am I missing an obvious problem?

Reply:
I think it is. 
I've tried it, but the problem is that the organizer's public key keeps being added to the list (because the code inside the component is executed again and again). I don't know how to solve this, I've tried many ways but neither of them worked. Since it might take some time to fix, I suggest we keep it as it is because it works fine. We can tackle this later or in another PR if that's ok for you.
We're now storing Public Keys instead of strings though.

Reply:
Actually it's not. Having PublicKeys instead of strings generate a problem because it seems that you can add them multiple times (neither of them are equal). So we'll keep strings.
But I was able to add the organizer's token directly when we open a roll call as you described.